content/en/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -273,33 +273,33 @@ Renders to:
 
 ## Source code files
 
-You can use the `{{%/* code */%}}` shortcode to embed the contents of file in a code block to allow users to download or copy its content to their clipboard. This shortcode is used when the contents of the sample file is generic and reusable, and you want the users to try it out themselves.
+You can use the `{{%/* code_sample */%}}` shortcode to embed the contents of file in a code block to allow users to download or copy its content to their clipboard. This shortcode is used when the contents of the sample file is generic and reusable, and you want the users to try it out themselves.
 
 This shortcode takes in two named parameters: `language` and `file`. The mandatory parameter `file` is used to specify the path to the file being displayed. The optional parameter `language` is used to specify the programming language of the file. If the `language` parameter is not provided, the shortcode will attempt to guess the language based on the file extension.
 
 For example:
 
 ```none
-{{%/* code language="yaml" file="application/deployment-scale.yaml" */%}}
+{{%/* code_sample language="yaml" file="application/deployment-scale.yaml" */%}}
 ```
 
 The output is:
 
-{{% code language="yaml" file="application/deployment-scale.yaml" %}}
+{{% code_sample language="yaml" file="application/deployment-scale.yaml" %}}
 
 When adding a new sample file, such as a YAML file, create the file in one of the `<LANG>/examples/` subdirectories where `<LANG>` is the language for the page. In the markdown of your page, use the `code` shortcode:
 
 ```none
-{{%/* code file="<RELATIVE-PATH>/example-yaml>" */%}}
+{{%/* code_sample file="<RELATIVE-PATH>/example-yaml>" */%}}
 ```
 where `<RELATIVE-PATH>` is the path to the sample file to include, relative to the `examples` directory. The following shortcode references a YAML file located at `/content/en/examples/configmap/configmaps.yaml`.
 
 ```none
-{{%/* code file="configmap/configmaps.yaml" */%}}
+{{%/* code_sample file="configmap/configmaps.yaml" */%}}
 ```
 
-The legacy `{{%/* codenew */%}}` shortcode is being replaced by `{{%/* code */%}}`.
-Use `{{%/* code */%}}` in new documentation.
+The legacy `{{%/* codenew */%}}` shortcode is being replaced by `{{%/* code_sample */%}}`.
+Use `{{%/* code_sample */%}}` in new documentation.
```

Comment:
```suggestion
The legacy `{{%/* codenew */%}}` shortcode is being replaced by `{{%/* code_sample */%}}`.
Use `{{%/* code_sample */%}}` (not `{{%/* code */%}}`) in new documentation.
```

Reply:
It is indeed better to clarify. Updated. Thanks.